packages/client/src/schema-component/antd/form-item/FormItem.Settings.tsx

```
@@ -592,6 +593,83 @@ export const formItemSettings = new SchemaSettings({
+                ...fieldSchema['x-component-props'],
+                fieldNames: {
+                  ...fieldSchema['x-component-props']?.['fieldNames'],
+                  formula,
+                },
+                appends: [fieldSchema['name']],
+              };
+              const regex = /{{(.*?)}}/g;
+              let match;
+              while ((match = regex.exec(formula))) {
+                if (match[1].includes('.')) {
+                  const matchList = match[1].split('.');
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)